.devcontainer/Dockerfile

```diff
@@ -17,10 +17,60 @@ RUN mkdir -p ${DENO_INSTALL} \
 
 ENV PATH=${DENO_INSTALL}/bin:${PATH} \
     DENO_DIR=/home/vscode/.cache/deno
+ENV PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium
+
+# Ripped from https://github.com/lucacasonato/deno-puppeteer/blob/main/Dockerfile
+RUN apt-get -qq update \
+    && apt-get -qq install -y --no-install-recommends \
+    curl \
+    ca-certificates \
+    unzip \
+    fonts-liberation \
+    libappindicator3-1 \
```

Comment:
If possible, I think we should have a docker image with the minimum set of packages so it doesn't get bigger and bigger.

Also, I'd suggest to merge `RUN` instructions to minimize created layers

Reply:
Aye to the 2nd comment. For the first one does it really matter? Since its just running a dev environment anyways

Reply:
it would be to ease the experience for new contributors